CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 ## [Unreleased]
 
+### Added
+- Added vendor scoping plugin to composer build `typisttech/imposter-plugin`.
+
+### Changed
+- Switching from Classmap to PSR-4 naming.
```

Comment:
Just a note, this is a super breaking change, so the next version should be 5.0.0

Reply:
Yes I have created a new milestone.